src/dbus.hpp

```diff
@@ -35,6 +35,12 @@ constexpr const char* busName = "com.yadro.Storage";
 constexpr const char* path = "/com/yadro/storage";
 } // namespace stormgr
 
+namespace netadpmgr
+{
+constexpr const char* busName = "com.yadro.NetworkAdapter";
+constexpr const char* path = "/com/yadro/network/adapter";
```

Comment:
I think, /com/yadro/network would be enough.

Reply:
I believe the network is a too generic name and it aggregates too wide a definition. We should care about that and I'd prefer to explicitly define `network-adapter`, not general `network`.

Reply:
Then call it `network_adapter`. I don't like an idea to add unneeded hierarchy level.

Reply:
And what do you think about the case when in the future, for example, we will be managing the host `network-vlan`s? 
By your design, that will be `network_vlan`. But that both things are network origin.
I believe the second level of the hierarchy is a good point there

Reply:
Ok.